src/server/multi_test.cc

```diff
@@ -627,6 +612,41 @@ TEST_F(MultiTest, GlobalFallback) {
   EXPECT_EQ(1, GetMetrics().ooo_tx_transaction_cnt);
 }
 
+TEST_F(MultiTest, ScriptConfigTest) {
+  if (auto config = absl::GetFlag(FLAGS_default_script_config); config != "") {
+    LOG(WARNING) << "Skipped Eval test because default_script_config is set";
+    return;
+  }
+
```

Comment:
There are no unit tests that change the `default_script_config` flag, because its read on startup, but the integration tests do (with eval_test)